README.windows.rst

```diff
@@ -41,9 +41,262 @@ Current status
 
+Ceph filesystems can be mounted using the ``ceph-dokan`` command, which
+requires the Dokany package to be installed. Note that dokany is a well
+maintained fork of the Dokan project, allowing filesystems to be implemented
+in userspace, pretty much like Fuse.
+
+RBD images can be mounted using the ``rbd`` or ``rbd-wnbd`` commands. The
+``WNBD`` driver is required for mapping RBD images on Windows.
+
 The libraries have to be built statically at the moment. The reason is that
 there are a few circular library dependencies or unspecified dependencies,
 which isn't supported when building DLLs. This mostly affects ``cls`` libraries.
 
-A significant number of tests from the ``tests`` directory have been port,
+A significant number of tests from the ``tests`` directory have been ported,
 providing adequate coverage.
+
+Supported platforms
+===================
+
+Windows Server 2019 and Windows Server 2016 are supported. Previous Windows
+Server versions, including Windows client versions such as Windows 10, might
+work but haven't been tested.
+
+Windows Server 2016 does not provide unix sockets, in which case the Ceph admin
+socket feature will be unavailable.
+
+Compatibility
+=============
+
+RBD images can be exposed to the OS and host Windows partitions or they can be
+attached to Hyper-V VMs in the same way as iSCSI disks.
+
+At the moment, the Microsoft Failover Cluster refuses to use WNBD disks as
+Cluster Shared Volumes (CSVs) underlying storage.
+
+OpenStack integration has been proposed as well and will most probably be
+included in the next OpenStack release, allowing RBD images managed by OpenStack
+Cinder to be attached to Hyper-V VMs managed by OpenStack Nova.
+
+.. _installing:
+Installing
+----------
+
+Soon we're going to provide an MSI installed for Ceph. For now, unzip the
+binaries that you may have obtained by following the building_ step.
+
+You may want to update the environment PATH variable, including the Ceph
+path. Assuming that you've copied the Ceph binaries to ``C:\Ceph``, you may
+use the following Powershell command:
+
+.. code:: bash
+
+    [Environment]::SetEnvironmentVariable("Path", "$env:PATH;C:\ceph", "Machine")
+
+In order to mount Ceph filesystems, you will have to install Dokany.
+You may fetch the installer as well as the source code from the Dokany
+Github repository: https://github.com/dokan-dev/dokany/releases
+
+Make sure to use 1.3.1, which at time of the writing is the latest
+stable release.
+
+In order to map RBD images, the ``WNBD`` driver must be installed. Please
+check out this page for more details about ``WNBD`` and the install process:
+https://github.com/cloudbase/wnbd
+
+Configuring
+-----------
+
+ceph.conf
+=========
+
+The default location for the ``ceph.conf`` file on Windows is
+``%ProgramData%\ceph\ceph.conf``, which usually expands to
+``C:\ProgramData\ceph\ceph.conf``. (Note - Directories with spaces
+in their names are not currently supported.)
```

Comment:
Nit: any more background on this limitation?

Reply:
It was mostly because of this: https://github.com/ceph/ceph/pull/32777/commits/ee7cfe916f9859eeb09189e124013bf2789cd44a

I think we can remove this note but I'll double check just to make sure.